src/ad-bidders/prebid/adapters/one-video.ts

```diff
@@ -0,0 +1,38 @@
+import { PrebidAdapter } from '../prebid-adapter';
+
+export class OneVideo extends PrebidAdapter {
+	static bidderName = 'oneVideo';
+
+	get bidderName(): string {
+		return OneVideo.bidderName;
+	}
+
+	prepareConfigForAdUnit(code, { site, pubId }): PrebidAdUnit {
+		return {
+			code,
+			mediaTypes: {
+				video: {
+					playerSize: [640, 480],
+					context: 'instream',
+				},
+			},
+			bids: [
+				{
+					bidder: this.bidderName,
+					params: {
+						site,
+						pubId,
+						video: {
+							playerWidth: 640,
+							playerHeight: 480,
+							mimes: ['video/mp4', 'application/javascript'],
+							protocols: [2, 5],
```

Comment:
```suggestion
							protocols: [2, 3, 5, 6],
```
It allows us to support VAST 3.0 protocol.
(numbers are  given in [pubmatic docs](http://prebid.org/dev-docs/bidders/pubmatic.html), search for `video.protocols`)